src/components/DrawerMain.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+  Box,
+  Button,
+  DialogContent,
+  DialogTitle,
+  IconButton,
+  ModalClose,
+  Sheet,
+} from "@mui/joy";
+import Drawer from "@mui/joy/Drawer";
+import Menu from "@mui/icons-material/Menu";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import SoundSlide from "./SoundSlide";
```

Comment:
The same. Please use absolute path instead relative (i.e. @app)

Reply:
Got it!